### PR TITLE
Issue #1004 - Adding Log Export Feature

### PR DESCRIPTION
**Pull request by adithyaphilip**
With reference to: https://github.com/open-keychain/open-keychain/issues/1004

The Export Log feature has been added to LogDisplayFragment.

### CODE REVIEW

File: OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/LogDisplayFragment.java [hunk 1]
```diff
@@ -70,6 +88,166 @@ public void onActivityCreated(Bundle savedInstanceState) {
...
+        }
+        if(pw!=null) {
+            pw.close();
+            if(!error && pw.checkError()) {//check if it is only pw.close() which generated error
+                currLog.add(OperationResult.LogType.MSG_EXPORT_LOG_EXPORT_ERROR_WRITING,1);
+                error = true;
+            }
+        }
+
+        if(!error)  currLog.add(OperationResult.LogType.MSG_EXPORT_LOG_EXPORT_SUCCESS,1);
```
Comment (Valodim) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/LogDisplayFragment.java]: code style

**Comment by Valodim** [comment]
Hi!

Looks good! Resolve that one code style thing and I'll gladly merge :+1:

For the future, you can avoid merge commits if you use `git pull --rebase` instead of `git pull`

**Comment by adithyaphilip** [comment]
Sorry, I've changed the code style, hope it's better now.

And thank you! I'm afraid I didn't know that. I'll make sure I do it henceforth :)

**Comment by dschuermann** [comment]
Thanks! Has been merged!

**Pull request merged**